app/views/admin/shelters/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render partial: "layouts/shelter_render" %>
+
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
+    <h3><%= shelter.name %></h3>
+    <p>Created at: <%= shelter.created_at %></p>
+    <%= link_to "Update #{shelter.name}", "/shelters/#{shelter.id}/edit" %>
+    <%= link_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
```

Comment:
This won't work without disabling Turbo! Remember that if you use a link_to for anything other than a GET request, Turbo will interfere. Be sure to include `{ turbo_method: :delete }` or just use a button instead